security.rst

```diff
@@ -4,144 +4,259 @@
 Security
 ========
 
-.. admonition:: Screencast
-    :class: screencast
+Symfony provides many tools to secure your application. Some HTTP-related
+security tools, like :ref:`secure cookies` and :ref:`CSRF protection` are
+provided by default. The SymfonySecurityBundle, which you will learn about
+in this guide, provides all authentication and authorization features
+needed to secure (parts of) your application.
 
-    Do you prefer video tutorials? Check out the `Symfony Security screencast series`_.
+.. _security-installation:
 
-Symfony's security system is incredibly powerful, but it can also be confusing
-to set up. Don't worry! In this article, you'll learn how to set up your app's
-security system step-by-step:
+To get started, make sure you have installed the SecurityBundle:
 
-#. :ref:`Installing security support <security-installation>`;
+.. code-block:: terminal
 
-#. :ref:`Create your User Class <create-user-class>`;
+    $ composer require symfony/security-bundle
 
-#. :ref:`Authentication & Firewalls <security-yaml-firewalls>`;
+If you have :ref:`Symfony Flex <symfony-flex>` installed, this also
+installs a ``security.yaml`` configuration file for you:
```

Comment:
`installs` -> `creates`

Reply:
Agreed, that's probably what the user experiences (although technically, Flex only installs files from the recipe)